addons/mail/wizard/mail_compose_message_views.xml

```diff
@@ -79,7 +79,7 @@
                         <button string="Log" name="action_send_mail"
                                 type="object" class="btn-primary" data-hotkey="q"
                                 attrs="{'invisible': [('subtype_is_log', '=', False)]}"/>
-                        <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="z" />
+                        <button name="cancel_composer" string="Cancel" class="btn-secondary" special="cancel" data-hotkey="z"/>
```

Comment:
Why this change ? All wizards of Odoo anyway when you cancel them ... why would we want another behavior here ? We globally never do that, as it adds unnecessary clicks to the user. You know what you do when you click on "cancel". Did you speak with @LNAodoo for this one, did he agree with adding clicks ?

Reply:
Hey, this task is essentially a way to prevent accidental closing of full composer.

This is still in RD Dev. The code is matching the specs, which cover pressing ESC and also the buttons (X and Cancel).
We can ask our PR @paju-odoo for not considering X/Cancel buttons.

Reply:
I can read the code, thanks ^^

What I say is that it makes no sense, as the button tells you what is about to happen. Note that you may rename "Cancel" to "Discard" if it seems clearer. But we generally fight to remove useless clicks, and it seems we are adding clicks where it is not necessary, especially we don't do that anywhere else.